src/scripts/main.js

```diff
@@ -1,3 +1,35 @@
 'use strict';
 
-// write your code here
+const promise1 = new Promise((resolve, reject) => {
+  const logo = document.querySelector('logo');
```

Comment:
The query selector `'logo'` is incorrect. It should be `'.logo'` if you are selecting by class, or `'#logo'` if selecting by ID. Ensure the selector matches the actual HTML element.